unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs

```diff
@@ -134,13 +134,18 @@ public static Vector2Int StringToVector2Int(string coords)
         }
 
         private const int MAX_TRANSFORM_VALUE = 10000;
+        private const float MIN_TRANSFORM_SCALE_Z_VALUE = 0.00001f;
 
         public static void CapGlobalValuesToMax(this Transform transform)
         {
             bool positionOutsideBoundaries = transform.position.sqrMagnitude > MAX_TRANSFORM_VALUE * MAX_TRANSFORM_VALUE;
             bool scaleOutsideBoundaries = transform.lossyScale.sqrMagnitude > MAX_TRANSFORM_VALUE * MAX_TRANSFORM_VALUE;
 
-            if (positionOutsideBoundaries || scaleOutsideBoundaries)
+            //Extra check to prevent rendering artifacts when bloom is on
+            //More description: https://github.com/decentraland/unity-renderer/issues/5698
+            bool scaleZAxisBelowMinimum = Mathf.Abs(transform.localScale.z) < MIN_TRANSFORM_SCALE_Z_VALUE;
```

Comment:
We saw the problem with Z axis but it sounds to me like it could perfectly happen with X or Y in 0 as well, maybe we should just have a generic `MIN_TRANSFORM_SCALE_VALUE` and check the 3 axis instead of just Z, what do you think?

Reply:
Yeah, I was thinking about this at the beginning, but when testing noticed that if you set scale on x or y the object is hidden and not being rendered at all. Because of that, I went with just z scale to avoid more breaking changes if creators are using scale in some way